packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+
+## Recent Changes
+
+- Enhancement: Inconsistent capitalization with z/OS console command. [Issue 961](https://github.com/zowe/zowe-cli/issues/961)
```

Comment:
Should wthis be a BugFix instead of an Enhancement?

Reply:
Probably, I believe that is the category we've used previously for typo-level changes.

Reply:
Shall I update it to BugFix ?

Reply:
Yes please 🙂

Reply:
Thank you!  😋